tornado/web.py

```diff
@@ -1803,20 +1803,15 @@ def render(*args, **kwargs) -> str:  # type: ignore
     def _ui_method(self, method: Callable[..., str]) -> Callable[..., str]:
         return lambda *args, **kwargs: method(self, *args, **kwargs)
 
-    def _clear_headers_for_304(self) -> None:
-        # 304 responses should not contain entity headers (defined in
-        # http://www.w3.org/Protocols/rfc2616/rfc2616-sec7.html#sec7.1)
+    def _clear_representation_headers(self) -> None:
+        # 304 responses should not representation metadata headers (defined in
```

Comment:
... shoud not **contain** representation metadata headers...

Reply:
Thanks, fixed, now with passing CI.